### PR TITLE
(rfc) updating rfc to reflect feedback and updated scope

### PR DESCRIPTION
**Pull request by JohnMahowald-minted**

### CODE REVIEW

File: the_mom_papers/the_mom_papers.md [hunk 1]
```diff
@@ -95,32 +94,55 @@ Not included in the current scope are:
 - [ ] Develop a pattern for checkout with non-customizable, downlodable skus
 - [ ] Build a tool that can scale to support increased variety of skus and high volume of traffic.
```
Comment (madeline-minted) [inline the_mom_papers/the_mom_papers.md]: what tool does this refer to?

File: the_mom_papers/the_mom_papers.md [hunk 2]
```diff
@@ -14,32 +14,34 @@ As a minted customer, I can...
...
+* As a customer, my downloadable wallpaper order is in the order status bar: 5 hours
+* As a customer, my digital order is visible in my-orders page: 5 hours
 
-#### Phase 2 (33 hours):
+#### Phase 2 (34 hours):
 **Download and Instrumetation**
-* As a customer, I have an authenticated dashboard for downloading digital wallpapers: 15 hours
-* As a customer, I can click on a button in my dashboard to download my wallpaper: 15 hours
+* As a customer, I have an order details page : 10 hours
+* As a customer, I have an /orders page: 6 hours
```
Comment (madeline-minted) [inline the_mom_papers/the_mom_papers.md]: what work are you anticipating to for these two ("As a customer, I have an order details page" and "As a customer, I have an /orders page")?

File: the_mom_papers/the_mom_papers.md [hunk 3]
```diff
@@ -49,15 +51,12 @@ Invitation to edit:
...
+**Tim Worsley and Adeel Khurshid, please review**
 
 Architect Signoff required
 - [ ] *Duplicating or forking existing code*: We will be extending some logic in the cart, orders, and mbo to allow for non-customizable, downloadable products.
 - [ ] *Introducing a new service endpoint*: We will expose an authenticated endpoint allowing users to come to the minted site and get a link to download their product.
 
-DBA Signoff is required
-- [ ] *Adds a new database or database table*: See Below
-
 DevOps/Security Signoff is required
```
Comment (AdeelKhurshid-minted) [inline the_mom_papers/the_mom_papers.md]: Signing off. no concerns with the aws usage or security.

Comment (JohnMahowald-minted) [inline the_mom_papers/the_mom_papers.md]: thanks @AdeelKhurshid-minted!

**Pull request merged**